README.md

```diff
@@ -56,6 +56,7 @@ _Handlers enriching log records._
 
 _Handlers forwarding logs._
 
+- [slog-clickhouse](https://github.com/smallnest/slog-clickhouse): Handler forwarding logs to ClickHouse.
```

Comment:
Please add a new line in this file: https://github.com/go-slog/awesome-slog/blob/main/data.yaml

Reply:
done